lib/MainNative/src/main.cpp

```diff
@@ -263,6 +265,10 @@ static int handleCommandLineArguments(PrgArguments& prgArguments, int argc, char
             {
                 prgArguments.serialTxChannel = optarg;
             }
+            else if (0 == strcmp(LONG_OPTIONS[optionIndex].name, "cwd"))
+            {
+                chdir(optarg);
```

Comment:
Handle it like for others too. Actions derived from it shall be done outside of this function.